server/dates.ts

```diff
@@ -30,7 +30,7 @@ export const postDate = async (
     return res.status(401).send({ message: 'User is not an admin' });
   }
 
-  const newDate: string = JSON.stringify(new Date(req.body.date));
+  const newDate: string = new Date(req.body.date).toISOString();
```

Comment:
`toISOString` converts dates to GMT. On the portal, we want the dates to be set as EST.

Reply:
the dates are stored in the database as GMT, but the date picker and selected date both default to the browser's local timezone (EST for most of us), and the storage in GMT are updated accordingly to be in line with the EST configured times. i saw online that its better to store as GMT in a database, but could switch this if you think it'd be beneficial!

Reply:
^^ never mind this, updated to store in EST because it wasnt working properly in the deployed version